communitycall-data.yml

```
@@ -40,3 +40,9 @@
     "2024-04-25": "mu"
     "2024-08-08": "js"
     "2024-09-19": "fkr"
+    "2024-10-10": "kg"
+    "2024-10-17": "js"
+    "2024-10-31": "kg"
+    "2024-11-09": "mu"
```

```suggestion
    "2024-11-07": "mu"
```
As discussed in Daily, this should be 2024-11-07